### PR TITLE
♻️ Dataservice study generator

### LINKED ISSUE #604: Data Service study generator for testing
We need to implement a study generator that produces fake realistic studies in the Data Service. 

It is very difficult to develop and test #502 with real studies in the Data Service since those studies are fairly large. Additionally, we need a more reliable way to test the ingest GWO task (and future ingest related functionality) without mocking out all of the responses from the Data Service.

### PR DESCRIPTION
**Pull request by znatty22**
Closes #604 

### Todo
- Add integration tests to CircleCI (maybe don't need parallel if dataservice images don't take long to build)
~- Replace placeholder docstrings in ingest package components~

### CODE REVIEW

File: creator/ingest_runs/data_generator/study_generator.py [hunk 1]
```diff
@@ -435,48 +467,102 @@ def workflow_type(row):
...
+            assert total == count, (
+                f"{endpoint} expected count {count} != {total} found"
+            )
 
-if __name__ == '__main__':
-    sg = StudyGenerator()
-    sg.ingest_study(initialize=False, create_files=False)
-    # sg.generate_files(create=True)
+
+def delete_entities(dataservice_url, study_id=None, logger=None):
```
Comment (znatty22) [inline creator/ingest_runs/data_generator/study_generator.py:516]: Should probably be part of the `kf-utils-python` repo?

File: creator/ingest_runs/data_generator/study_generator.py [hunk 2]
```diff
@@ -1,162 +1,267 @@
...
 
-1. Create (or read previously created) study files
-- Create a biological TSV file with minimal family, participant, specimen data
-- Create source and harmonized genomic files
-- Upload genomic files to study bucket on S3
-- Create S3 object manifest for source genomic files
-- Create the BIX Genomic Workflow Output Manifest
-
-2. Ingest study files into the KF Data Service (except the BIX manifest)
+1. (Optional) Delete all previously data on disk, delete study in Data Service
```
Comment (gsantia) [inline creator/ingest_runs/data_generator/study_generator.py:5]: `1. (Optional) Delete all previously generated data on disk, delete study in Data Service`

File: creator/ingest_runs/data_generator/study_generator.py (hunk 1 above)
Comment (znatty22) [reply creator/ingest_runs/data_generator/study_generator.py:516]: Unfortunately kf-utils-python doesn't have any testing setup so we'll probably have to leave this here for now

**Review by gsantia — APPROVED** [review]
🤠

**Pull request merged**